src/components/Pillar/Pillar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default function Pillar({
+  count,
+  label,
+}: {
+  count: string;
+  label: string;
+}) {
+  return (
+    <div className="flex max-w-xs md:max-w-sm space-x-4 p-4 border rounded-lg items-center md:bg-[#09AFC6]">
+      <h1 className="text-2xl md:text-4xl font-extrabold md:text-white tracking-wide">
+        {count}
+      </h1>
```

Comment:
Semantically, we should not use a heading tag here. Screen readers will announce this as a heading and we do not want that. Let's change it to a `p` tag instead.

Reply:
The necessary changes have been made.